sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -101,6 +103,9 @@ private void InitilizeBlobBindings(ExtensionConfigContext context)
             rule.BindToInput<MultiBlobContext>(this); // Intermediate private context to capture state
             rule.AddOpenConverter<MultiBlobContext, IEnumerable<BlobCollectionType>>(typeof(BlobCollectionConverter<>), this);
 
+            // Bind to ParameterBindingData
+            rule.BindToInput<ParameterBindingData>((attr) => CreateBindingData(attr)); // Precedence, must beat BindToStream
```

Comment:
Need to add some tests that exercise/verify this new binding. There's some e2e binding test suites, e.g. https://github.com/Azure/azure-sdk-for-net/blob/fce7915180ccab24533014e1dea89e89fa6d2e90/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/tests/BlobTriggerTests.cs#L19